mlrun/config.py

```diff
@@ -541,6 +541,17 @@
         # interval for stopping log collection for runs which are in a terminal state
         "stop_logs_interval": 3600,
     },
+    # Configurations for the `mlrun.package` sub-package involving packagers - logging returned outputs and parsing
+    # inputs data items:
+    "packagers": {
+        # Whether to enable packagers. True will wrap each run in the `mlrun.package.handler` decorator to log and parse
+        # using packagers.
+        "enabled": True,
+        # Whether to treat returned tuples from functions as a tuple and not as multiple returned items. If True, all
+        # returned values will be packaged together as the tuple they are returned in. Default is False to enable
+        # logging multiple returned items.
+        "pack_tuples": False,
```

Comment:
it is synced. If you mark it as True you must provide 1 log hint, otherwise the other would be ignored... It makes sense. The use-case is someone who return a tuple and want to log it. Not common as usually you return lists but I made it as a conf option to cover all possibilities. I can delete it as well if you think it is redundant :)